packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -46,6 +46,31 @@ const postCSSLoaderOptions = {
   ],
 };
 
+// style files regexes
+const CSSRegex = /\.css$/;
+const CSSModuleRegex = /\.module\.css$/;
+const SASSRegex = /\.(scss|sass)$/;
+const SASSModuleRegex = /\.module\.(scss|sass)$/;
+
+// Common function to create any style loader
+const getStyleLoader = (
+  postStyleLoader,
```

Comment:
What if we just called it `loaderName` and `loaderOptions`? Maybe I'm being too nit-picky. Anyone else have an opinion?

Reply:
I've refactored that running also `postcss`, which made things simpler, you might check that again!